contracts/BalancerYBALVoter.sol

```diff
@@ -66,10 +61,12 @@ contract BalancerYBALVoter {
         governance = _governance;
     }
     
-    function execute(address to, uint value, bytes calldata data) external returns (bool, bytes memory) {
+    function execute(address payable to, uint value, bytes calldata data) external returns (bool, bytes memory) {
```

Comment:
function needs to be payable aswell if you want to send ETH through the function. If msg.value is always 0, and the ether is sent directly from the contract then this can work. However, if you want msg.value to be part of the ether sent you need to make the function payable.

`function execute(address payable to, uint value, bytes calldata data) external payable returns (bool, bytes memory)`